contracts/jobs/JobsManager.sol

```diff
@@ -527,9 +492,9 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         uint256 blockNum = roundsManager().blockNum();
         uint256 challengeBlock = claim.claimBlock + 1;
         // Must be after verification period
-        require(blockNum > claim.endVerificationBlock);
+        require(blockNum >= claim.endVerificationBlock);
```

Comment:
Treating the `endVerificationBlock` as the block in which the verification period is over